packaging/cmake/try_compile/pthread_setname_darwin_test.c

```diff
@@ -0,0 +1,6 @@
+#include <pthread.h>
```

Comment:
and here

Reply:
Done, but again put the define in the CMake file for congruency with the GNU check.